src/OSPSuite.Utility/OSPSuite.Utility.csproj

```diff
@@ -1,214 +1,38 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
Look at that!! everything is gone . The csproj file is SO MUCH EASIER to follow!